pkg/sql/logictest/testdata/logic_test/computed

```diff
@@ -758,9 +758,12 @@ CREATE TABLE x (
   b INT AS (a+1) STORED
 )
 
-query error value type decimal doesn't match type int of column "a"
+query error unsupported binary operator: <decimal> \+ <int> \(desired <int>\)
 INSERT INTO x VALUES(1.4)
```

Comment:
Postgres allows this by rounding the decimal to an int, so once we add assignment casts it should not be an error, correct?

Reply:
yep, if you put the cast type as assignment.